backbone/src/main/java/org/researchstack/backbone/step/AudioTooLoudStep.java

```diff
@@ -0,0 +1,95 @@
+package org.researchstack.backbone.step;
+
+import org.researchstack.backbone.result.AudioResult;
+import org.researchstack.backbone.result.StepResult;
+import org.researchstack.backbone.result.TaskResult;
+import org.researchstack.backbone.task.NavigableOrderedTask;
+import org.researchstack.backbone.utils.LogExt;
+import org.researchstack.backbone.utils.StepResultHelper;
+
+import java.util.List;
+
+/**
+ * Created by TheMDP on 2/26/17.
+ *
+ * The AudioTooLoudStep is designed to be placed after an ActiveStep with identifier audioStepResultIdentifier
+ * The ActiveStep should also have an AudioRecorder that produces an AudioResult that can be
+ * analyzed in this state to see if this should show to the user that their background noise
+ * and the recording environment they are in, is too loud, and therefore unsatisfactory for
+ * the proceeding ActiveStep with AudioRecorder to collect accurate data
+ */
+
+public class AudioTooLoudStep extends InstructionStep implements NavigableOrderedTask.NavigationSkipRule {
+
+    /**
+     * The step identifier where the AudioResult object will be
+     */
+    private String audioStepResultIdentifier;
+
+    /**
+     * Must be a value from 0.0 - 1.0
+     * Step will be skipped unless the loudness threshold is over this threshold
```

Comment:
Oh, actually this is correct.  We always skip this step, which says, "noise too loud", unless we want to show it when the loudness is over the threshold.